### ISSUE TITLE
Промотка к ссылкам при клике на таймкод

### ISSUE THREAD

**Issue by inoyakaigor** [web-standards-ru/web-standards.ru#365]
…очень неимоверно бесит потому, что приходится каждый раз обратно отматывать наверх

**Comment by sugrarin** [comment]
А зачем наверх отматывать?

**Comment by inoyakaigor** [comment]
@sugrarin потому, что на странице таймкоды кликабельны и я ими пользуюсь, что промотать к следующей теме

**Comment by sugrarin** [comment]
[@inoyakaigor](https://github.com/inoyakaigor) ну так и зачем отматывать наверх-то? Таймкоды остаются на виду при прокрутке к ссылкам. 

Может бесит не прокрутка к ссылкам, а то, что тогда таймкоды пропадают на мобиле?

**Comment by inoyakaigor** [comment]
@sugrarin да, наверное, ты прав. Бесит не сама промотка, а то, что таймкоды улетают вверх за пределы экрана
